src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -42,7 +42,8 @@ public JsonAdaptedPerson(
             @JsonProperty("phone") String phone,
             @JsonProperty("email") String email,
             @JsonProperty("internshipId") Integer internshipId,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+            @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
+            @JsonProperty("linkIndex") Integer linkIndex) {
```

Comment:
I think we can remove this line since `Person` does not have a `linkIndex` field

Reply:
Similarly, we can remove all the lines in `JsonAdaptedPersonTest` with `VALID_LINK_INDEX` in them